modules/programs/rofi.nix

```diff
@@ -405,7 +416,10 @@ in {
       inherit value;
     };
 
-    home.packages = [ cfg.package ];
+    home.packages = let
+      rofiWithPlugins = cfg.package.override
```

Comment:
I think the tests are failing on this because of the overlays. Any advice @berbiche or @rycee on how to handle this situation?

Reply:
Yes, I have tested it without overlays and works fine. Also, what is the point of those overlays with empty `writeScriptBin`?

Reply:
The point is so that you don't have to download the actual binary in order to run the tests.